src/components/molecules/Button/styles.module.scss

```diff
@@ -0,0 +1,117 @@
+@import 'src/styles/variables';
+@import 'src/styles/mixins';
+
+.baseButton {
+  display: flex;
+}
+
+.buttonText {
+  order: 2;
+  @include buttonTextStyle;
+  line-height: $line-height-24;
+}
+
+.btn {
+  $self: &;
+  appearance: none;
+  display: inline-flex;
+  align-items: center;
+  background: none;
+  border: 0;
+  color: $color-white;
+  cursor: pointer;
+  overflow: visible;
+  padding: 8px 40px;
+  text-decoration: none;
+  border-radius: $border-radius-20;
+  white-space: nowrap;
+  background-color: $color-primary;
+
+  &:focus {
+    outline: none;
+  }
+
+  &--disabled {
+    cursor: not-allowed;
+  }
+
+  &--primary {
+    background-color: $color-primary;
+
+    &:hover,
+    &:active,
+    &:focus {
+      background-color: $color-secondary-dark;
+    }
+
+    &:focus {
+      border: 2px solid $color-navigation-icon;
+    }
+
+    &#{$self}--disabled {
+      background-color: $color-background-disabled;
+      border: none;
+    }
+  }
+
+  &--secondary {
+    background-color: $color-white;
+    border: 2px solid $color-border-secondary;
+    color: $color-text;
+
+    &:hover,
+    &:active {
+      border: 2px solid $color-border;
+    }
+
+    &:focus {
+      border: 2px solid $color-primary;
+    }
+
+    &#{$self}--disabled {
+      color: $color-text-light;
+      background-color: $color-background;
+      border: 2px solid $color-background-disabled;
+    }
+  }
+
+  &--text {
+    padding: 0;
+    background: none;
+    color: $color-primary;
+    border-radius: 0;
+
+    &:hover,
+    &:active {
+      text-decoration: underline;
+    }
+
+    &:focus {
+      box-shadow: inset 0 0 0 2px $color-primary;
+    }
+
+    &#{$self}--disabled {
+      color: $color-text-light;
+      text-decoration: none;
+      box-shadow: none;
+    }
+  }
+
+  &--s {
+    padding: 5px 24px;
+    @include smallButtonTextStyle;
+  }
+}
+
+.icon {
```

Comment:
This will change, once we use svg directly anyway, but generally, when exporting icons from figma, then take the full 24px x 24px version. They all have a box around them and it will make it easier to deal with any icon. We will also not have any weird padding/width/height/margin values this way.

**[note]**

Extra note here is that we should use the "spacing" helper for all margin/padding/gap values to make sure alignment is correct everywhere.